comparators/invalid-tag-modification.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+module.exports = invalidTagModification;
+
+
+function getPrimaryTag(properties) {
+  var primaryTags = [
+    'aerialway', 'aeroway', 'amenity', 'barrier', 'boundary', 'building', 'craft', 'emergency',
+    'geological', 'highway', 'historic', 'landuse', 'leisure', 'man_made', 'military', 'natural',
+    'office', 'places', 'power', 'public_transport', 'railway', 'route', 'shop', 'sport',
+    'tourism', 'waterway'
+  ];
+  for (var key in properties) {
+    if (primaryTags.indexOf(key) !== -1) return key;
```

Comment:
@bkowshik Wondering if a feature has more than one primary tag( We already have a cf for check [bad primary tag combinations](https://github.com/mapbox/osm-compare/blob/master/comparators/invalid-tag-combination.js)). In that case, only the first tag which comes through the loop will be checked for modification. How would it be to return the set of primary tags instead?